src/plugin-store.svelte

```diff
@@ -110,18 +110,17 @@ let pluginStore: Element;
         bind:open={isOpen}
         fullscreen
         surface$style="width: 850px; max-width: calc(100vw - 32px);"
-        aria-labelledby="plugin-store-title"
+        aria-labelledby="plugin-store-header--title"
         aria-describedby="plugin-store-content"
     >
-        <Header>
-            <Title id="plugin-store-title">Plugin Store</Title>
-            <IconButton class="material-icons" action="close">
-                <IconClose />
-            </IconButton>
-        </Header>
-        <Content id="plugin-store-content">
-            <plugin-store bind:this={pluginStore}>
-                <plugin-store-toolbar>
+        <Header style="flex-direction: column; box-shadow: none;">
+            <plugin-store-header--top>
+                <Title id="plugin-store-header--title">Plugin Store</Title>
```

Comment:
what purpose have the `ids`?

Reply:
It's used by MUI's Dialog attributes: `aria-labelledby` and `aria-describedby`.